src/app/src/Screensaver.js

```diff
@@ -1,15 +1,37 @@
-import React from 'react';
-
-export default function Screensaver() {
-    return (
-        <div className='screensaver'>
-            <div className='screensaver__message'>
-                Learn how the fish ladder helps fish populations and whether you
-                have what it takes to be an aquatic biologist.
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { func } from 'prop-types';
+import { hideScreensaver } from './actions';
+
+class Screensaver extends Component {
+    render() {
+        const { dispatch } = this.props;
+
+        return (
+            <div className='screensaver'>
+                <div className='screensaver__message'>
+                    Learn how the fish ladder helps fish populations and whether
+                    you have what it takes to be an aquatic biologist.
+                </div>
+                <button
+                    type='button'
+                    className='screensaver__button'
+                    onClick={() => dispatch(hideScreensaver())}
```

Comment:
@alexelash might disagree, but I think it would be much better to have the screensaver be hidden in response to clicks anywhere on the screen, not just the "Let's Go" button.

The button is useful for a visual call-to-action, but I think hiding the screensaver only if you click that button might get quite frustrating. When observing the guided tour at the WaterWorks, the guide didn't even look at the screen when tapping it to take it out of screensaver mode.

Reply:
I do not disagree! Apologies for not noting anywhere.

Reply:
Good call! If I don't find a quick solution to this for this PR, I'll punt it to a new card. At least the action/reducers now exist.

Reply:
Addressed in 142f6c2